src/shared/js/ajax.html

```diff
@@ -247,13 +247,18 @@
 			return output;
 		},
 
-		fire: function(type, detail) {
-			if (this._eventPrefix) {
+		fire: function(type, detail, options) {
+			if (this._eventPrefix && DataUtils.getPathValue('prefix', options)) {
```

Comment:
Looks like this needs to be `StrandLib.DataUtils.getPathValue('prefix', options)`